conans/test/utils/tools.py

```diff
@@ -503,9 +503,14 @@ def run(self, command_line, user_io=None, ignore_error=False):
                 sys.modules.pop(added, None)
 
         if not ignore_error and error:
-            logger.error(self.user_io.out)
-            print(self.user_io.out)
```

Comment:
I've removed output to `logger` and `print`, I think that in a testing context those are not required, are they?

Reply:
The print is the thing that is useful when --nocapture, but yeah, overall it makes more sense the formatted Exception